homeassistant/components/openuv/__init__.py

```diff
@@ -118,6 +120,13 @@
     identifier = '{0}, {1}'.format(latitude, longitude)
 
     if identifier not in configured_instances(hass):
+        websession = aiohttp_client.async_get_clientsession(hass)
```

Comment:
This is not necessary. Validate it before saving it inside a config entry (so in the flow), and when setting up a config entry.

Reply:
Sorry, I'm not following. Isn't that what I'm doing? This prevents the config entry from being saved unless the API key is valid.